schema/buildings/defs.yaml

```diff
@@ -20,6 +20,11 @@ shapeContainer:
       description: >-
         The height of the bottom part of building in meters. Used if a building or part of building starts above the ground level.
       type: number
+    min_floor:
+      description: >-
+        The "start" floor of this building or part. This is the analog of "min_height" but is sometimes populated when the
```

Comment:
Is this split to multiple lines a problem ? It is different from other descriptions, that's why I am asking

Reply:
Not a problem - "usually" we wrap the lines around 72 or 80 though, although that's pretty inconsistent.